src/plugins/CaptionsPlugin.spec.js

```diff
@@ -33,4 +33,16 @@ describe('CaptionsPlugin', () => {
     expect(cp.getCaptionsStyles('font')).to.equal('arial');
     expect(cp.getCaptionsStyles()).to.be.instanceof(Object);
   });
+
+  it('Plugin should work without any controls', (done) => {
+    //set up empty plugin
+    cp = new CaptionsPlugin();
+    cp.preload({ client: new Bellhop() });
+    cp.init();
+    cp.client.trigger('features', {
+    });
+
+    done();
```

Comment:
These tests should not async if they are not actually waiting for a callback to fire.